packages/server/knexfile.js

```diff
@@ -86,14 +86,16 @@ const config = {
     connection: {
       connectionString: connectionUri || 'postgres://127.0.0.1/speckle2_test',
       application_name: 'speckle_server'
-    }
+    },
+    asyncStackTraces: true
```

Comment:
Can't believe we hadn't turned this on - this finally enables stack traces that properly tell you where the broken SQL query was triggered from

Reply:
This is already only enabled for dev & test envs. And on top of that they do themselves say that its a small overhead.